doc/sources/conf.py

```diff
@@ -60,15 +65,14 @@
 # other places throughout the built documents.
 #
 os.environ['KIVY_DOC_INCLUDE'] = '1'
-import kivy
```

Comment:
importing kivy [reads environment](https://github.com/kivy/kivy/blob/master/kivy/__init__.py#L214), which means you just broke the doc config. See what happens [next](https://github.com/kivy/kivy/blob/master/kivy/__init__.py#L269). 🔍